website/docs/guides/migration/versions/05-upgrading-to-v1.3.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Upgrading to v1.3 (latest)"
+title: "Upgrading to v1.3"
```

Comment:
Small thing: We leave "latest" here until v1.4 has its actual-factual final release, which will be in ~3 weeks